src/components/Header/components/SearchButton.tsx

```diff
@@ -6,24 +6,40 @@ import Button from "../../Button/Button";
 import Icon from "../../Icons/Icon";
 import SearchForm from "./SearchForm";
 
-const SearchButton = chakra(() => {
+interface SearchButtonProps {
+  isMobile?: boolean;
+}
+
+const SearchButton = chakra(({ isMobile = false }: SearchButtonProps) => {
   const [isOpen, setIsOpen] = useState<boolean>(false);
-  const styles = useStyleConfig("HeaderSearchButton", { isOpen });
+  const styles = useStyleConfig("HeaderSearchButton", { isMobile, isOpen });
+  const buttonText = isMobile ? null : isOpen ? "Close" : "Search";
+  const buttonTextColor = isOpen
+    ? "ui.white"
+    : isMobile
+    ? "ui.black"
+    : "ui.focus";
```

Comment:
I think because `ui.focus` looked more like the existing button color. But `link.primary` is not that far off so I'll switch.

I did that style here because of the specific `color` prop that exists for the `Icon` component. So now, if this prop exists because we defined it, should we use it or set the color in the theme file? For now, I'll move it over.